Session.js

```diff
@@ -0,0 +1,53 @@
+class Session { 
+// To be utilized when unwrapping JSON from the future servlets. Will be on the 
+// client side and be used as a buffer for client to server information needed  
+// for other components of Virtual Movie Night.
+ 
+    constructor() {
+        this.sessionId = expected.sessionId;
```

Comment:
you can get rid of the expected stuff, I was saying to do this in the fromObj (with the obj being passed in looking like expected, an example JSON from the servlet), which you did

Reply:
Ahhhh, gotcha